knative-operator/hack/update-deps.sh

```diff
@@ -1,24 +1,65 @@
 #!/usr/bin/env bash
 
 readonly ROOT_DIR=$(dirname $0)/..
+source ${ROOT_DIR}/vendor/knative.dev/test-infra/scripts/library.sh
 
 set -o errexit
 set -o nounset
 set -o pipefail
 
+cd ${ROOT_DIR}
+
+# This controls the knative release version we track.
+KN_VERSION="release-0.18"
+
+# Controls the version of OCP related dependencies.
+OCP_VERSION="release-4.6"
+
+# The list of dependencies that we track at HEAD and periodically
+# float forward in this repository.
+FLOATING_DEPS=(
+  "github.com/openshift/api@${OCP_VERSION}"
+
+  "knative.dev/operator@${KN_VERSION}"
+  "knative.dev/pkg@${KN_VERSION}"
+  "knative.dev/serving@${KN_VERSION}"
```

Comment:
How about `knative.dev/eventing-contrib` and `knative.dev/eventing`?

Reply:
Neither are required in `go.mod` (nor vendored)